.github/workflows/falke8.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       - name: Setup environment
         run: pipenv sync --dev
       - name: Run Flake8
-        run: pipenv run flake8 --max-line-length 120
+        run: pipenv run flake8 --max-line-length 120 --ignore=E111,W391,E303
```

Comment:
Why do you need to ignore these?
can you add a comment in the code?

Reply:
I could.. i just dont see any reason to leave them in as those errors are generally ignored.